### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Inspect][MAS4.2.1][Eclipse][Wipro]: Inappropriate control type for the Table in "Run on web app for containers dialog"

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1400]
**Repro Steps:**

1. Open and Start Eclipse.
2. Create a new dynamic web project.
3. Navigate to and select Project ->Azure->Publish to web app for containers
4. "Run on web app for containers" dialog will open. 
5. In "Run on web app for containers" dialog, tab to and select  "web app for container" button.->Use existing radio button
6. Tab to table and observe the control type using inspect.

**Actual Result:**
Control type for table is list.
Narrator table commands such as "Caps+F3", "Caps+F8" are working fine.

**Narrator :**
Table, has <say-as interpret-as="cardinal">13</say-as> rows, <say-as interpret-as="cardinal">2</say-as> columns,, java-webapp-5534, <say-as interpret-as="spell">3</say-as> of 13, selected, 

**NVDA:**
list
java-webapp-5534; Resource group: javacsmrg4397  3 of 13

**Expected Result:**
Control type should be Table.

**Platform:**
Eclipse Version:
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
OSBuild Version: Version 1709(Os Build 16299.192)
![run on web app for containers-table-inspect](https://user-images.githubusercontent.com/22956130/35684551-437ee2f0-078d-11e8-9f13-11a925302c6f.jpg)
![run on web app fro containers-table-narrator](https://user-images.githubusercontent.com/22956130/35684552-465512b0-078d-11e8-86bf-a5c207672d33.jpg)
![run on web app fro containers-table-nvda](https://user-images.githubusercontent.com/22956130/35684558-4a6a4406-078d-11e8-9fc8-8638bfb35c3f.jpg)

**Comment by Eskibear** [comment]
duplicated with #1405

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749837
